Sources/PortalSwift/Provider/PortalProvider.swift

```diff
@@ -20,7 +20,7 @@ public class PortalProvider {
 
   private let apiKey: String
   public var autoApprove: Bool
-  public var chainId: Chains.RawValue?
```

Comment:
Could this value be depended on as a type for other customers?

Reply:
the Chains.RawValue is Int btw, they cannot pass Chains to it. it should be Chains.RawValue which is Int at the end.